packages/web/components/EditProfileModal.tsx

```diff
@@ -175,6 +114,7 @@ export const EditProfileModal: React.FC<ProfileEditorProps> = ({
   onClose,
 }) => {
   const [status, setStatus] = useState<Maybe<ReactElement | string>>();
+
   // eslint-disable-next-line react-hooks/exhaustive-deps
   const username = useMemo(() => player?.profile?.username, []);
```

Comment:
Does this need to be memoized. It is just an object dereference for a string which will have referential integrity without the `useMemo`.
```suggestion
  const username = player?.profile?.username;
```